enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/CypherComparisonSupport.scala

```diff
@@ -492,53 +495,55 @@ object CypherComparisonSupport {
     def prepare(): Unit = newRuntimeMonitor.clear()
 
     def checkResultForSuccess(query: String, internalExecutionResult: InternalExecutionResult): Unit = {
-      val (reportedRuntime: String, reportedPlanner: String, reportedVersion: String) = extractConfiguration(internalExecutionResult)
-      // Rule planner only exists in 3.1 and earlier
-      val rulePlannerFallback = Planners.Rule.acceptedPlannerNames.contains(reportedPlanner) && Versions.V3_1.acceptedVersionNames.contains(reportedVersion)
-
+      val (reportedRuntime: String, reportedPlanner: String, reportedVersion: String, reportedPlannerVersion: String) = extractConfiguration(internalExecutionResult)
       if (!runtime.acceptedRuntimeNames.contains(reportedRuntime))
         fail(s"did not use ${runtime.acceptedRuntimeNames} runtime - instead $reportedRuntime was used. Scenario $name")
       if (!planner.acceptedPlannerNames.contains(reportedPlanner))
         fail(s"did not use ${planner.acceptedPlannerNames} planner - instead $reportedPlanner was used. Scenario $name")
-      if (!(version.acceptedVersionNames.contains(reportedVersion) || rulePlannerFallback))
-        fail(s"did not use ${version.acceptedVersionNames} version - instead $reportedVersion was used. Scenario $name")
+      if (!version.acceptedRuntimeVersionNames.contains(reportedVersion))
```

Comment:
Have we removed support for testing the RULE planner now?

Reply:
No, but we have removed rule fallback i.e. you can not write something like Cypher 3.3 planner=rule anymore so rule will not be a special case